packages/block-library/src/embed/variations.native.js

```diff
@@ -0,0 +1,3 @@
+const variations = [];
```

Comment:
Are we creating a native `variations` because we are supporting 8 services in the inserter instead of 37 like the web? Or is there another reason?

Reply:
That's exactly the reason 👍 When implementing [this issue](https://github.com/wordpress-mobile/gutenberg-mobile/issues/3278), will probably include all the variations here as well to have them recognized when pasting a new link as their URL patterns seem to also live in the `variations`. But will only show the 8 variations we want in the inserter by changing some of their scope to `inserter`.